### ISSUE TITLE
Accessibility, Talkback users still cannot answer calls from lock screen

### ISSUE THREAD

**Issue by Pummelo65** [BelledonneCommunications/linphone-android#964]
The talkback screenreader does not report the slider to answer a call in lock screen where mUseSliderMode is true preventing calls from being answered (or declined).
I see two solutions:
It is possible to detect if talkback (or any other possible screenreader) is running.
https://stackoverflow.com/questions/5116867/how-to-know-if-android-talkback-is-active
If this is the case mUseSliderMode could be set to false during lock screen allowing to answer to be a regular button click.

The probably better alternative would be to make the view accessible by itself:
https://developer.android.com/guide/topics/ui/accessibility/custom-views
assigning it the role of a slider.
I noted the commit at
https://gitlab.linphone.org/BC/public/linphone-android/commit/e7d67fdfeb7fb626e0aed06de89e9136710d66bf
however, Talkback still reports the answer sliders as buttons which they are not really.
Any thoughts?
/Roland

**Comment by Viish** [comment]
Hi Roland,
Thanks for the feedback.
I have to admit I'm no expert in accessibility services.
I've enabled the screen reader on my device and on the incoming call view while screen is locked, with two fingers I can slide the buttons. With the proper content description (saying slide to the right to decline the call, slide to the left to answer the call) is that enough or slide action is too complicated?
Best regards.

**Comment by Pummelo65** [comment]
Hi, 
> the proper content description (saying slide to the right to decline the call, slide to the left to answer the call) is that enough or slide action is too complicated?
No, such behavior would be absolutely ideal but on my Phone, Pixel, Android 10, this is not read and I will run the app in debugger and see if I can trace down what’s going on.
Thanks so much and best regards
Roland
